lib/grpc_kit/rpc_desc.rb

```diff
@@ -29,19 +29,37 @@ class RpcDesc
     # @param marshal_method [Symbol] method name of marshaling which marshal is called this method
     # @param unmarshal_method [Symbol] method name of unmarshaling which unmarshal is called this method
     # @param service_name [String]
-    def initialize(name:, marshal:, unmarshal:, marshal_method:, unmarshal_method:, service_name:)
+    # @param max_server_message_size [Integer, nil] Specify maximum size of server outbound message in bytes. Default to 4MB. Takes precedence over the value specified to GrpcKit::Server and GrpcKit::Client instance.
```

Comment:
s/server outbound/server inbound/ ?

Reply:
RpcDesc is used by both client and server side, so it should have the configuration from both aspects. See #build_server and #build_client - this value will be max_send_message_size on a server and max_receive_message_size on a client.

Reply:
Understood. Thanks for your detailed comment